### ISSUE TITLE
Obsolete interval parameter for all except intraday

### ISSUE THREAD

**Issue by danger89** [zackurben/alphavantage#80]
Hi,

Thanks for providing a JS wrapper around the API.

You are saying that your API calls for time series are like this:

```js
alpha.data.intraday(symbol, outputsize, datatype, interval);
alpha.data.daily(symbol, outputsize, datatype, interval);
alpha.data.daily_adjusted(symbol, outputsize, datatype, interval);
alpha.data.weekly(symbol, outputsize, datatype, interval);
alpha.data.weekly_adjusted(symbol, outputsize, datatype, interval);
alpha.data.monthly(symbol, outputsize, datatype, interval);
alpha.data.monthly_adjusted(symbol, outputsize, datatype, interval);
alpha.data.quote(symbol, outputsize, datatype, interval);
```

But the fact is the only `intraday` support `interval` parameter. The rest of the calls the `internal` is **obsolete** (and even an **invalid request**).

You could clean-up your API? And update your docs plz?
This is only very confusing for people.

- [Reference API doc of Alpha Vantage](https://www.alphavantage.co/documentation/#intraday)

Regards,
Melroy van den Berg

**Comment by zackurben** [comment]
@danger89 Thanks for the heads up, it looks like the API may have changed. I'll get it fixed.

**Comment by zackurben** [comment]
This was addressed in 2.1.0, thanks.
